Mathlib/Tactic/Peel.lean

```diff
@@ -3,8 +3,8 @@ Copyright (c) 2023 Jireh Loreaux. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jireh Loreaux
 -/
-import Mathlib.Order.Filter.Basic
 import Mathlib.Tactic.Basic
+import Mathlib.Order.Filter.Basic
```

Comment:
I'm not going to insist that you sort the import blocks, but let's at least not make it worse :)
```suggestion
import Mathlib.Order.Filter.Basic
import Mathlib.Tactic.Basic
```